src/EFCore.PG/Design/Internal/NpgsqlAnnotationCodeGenerator.cs

```diff
@@ -137,6 +137,8 @@ public override MethodCallCodeFragment GenerateFluentApi(IIndex index, IAnnotati
                 return new MethodCallCodeFragment(nameof(NpgsqlIndexBuilderExtensions.ForNpgsqlHasMethod), annotation.Value);
             if (annotation.Name == NpgsqlAnnotationNames.IndexOperators)
                 return new MethodCallCodeFragment(nameof(NpgsqlIndexBuilderExtensions.ForNpgsqlHasOperators), annotation.Value);
+            if (annotation.Name == NpgsqlAnnotationNames.IndexInclude)
```

Comment:
This is actually missing [from the SqlServer provider](https://github.com/aspnet/EntityFrameworkCore/blob/777a8a94ddc1f7780f9c943b079bd214686cb401/src/EFCore.SqlServer/Design/Internal/SqlServerAnnotationCodeGenerator.cs#L64), any particular reason as far as you know? It it's just an oversight consider sending them a PR to fix it...

Reply:
I have no clue...

Reply:
Sent them a PR; https://github.com/aspnet/EntityFrameworkCore/pull/13968 😄